docs/contact.html

```diff
@@ -108,11 +108,11 @@ <h2 id="">Social media</h2>
          </div>
          <div class="col-sm-5 twocol">
 
-<!--          <h2 id="newsletter">Newsletter</h2>-->
+          <h2 id="newsletter">Newsletter</h2>
 
-<!--          <p><i class="fa fa-envelope"></i>-->
-<!--             <a href="https://joyofcoding.us6.list-manage.com/subscribe?u=3c1a490ebd&id=a52a450c87">Sign up for our low-traffic newsletter</a> - roughly one e-mail per month in the half-year before the conference,-->
-<!--             or read the previous newsletters in our <a href="https://us6.campaign-archive.com/home/?u=3c1a490ebd&id=a52a450c87">archive</a>.</p>-->
+          <p id="subscribe"><i class="fa fa-envelope"></i>
```

Comment:
In plaats van een id zou je ook een class toe kunnen voegen voor om dit soort tekst blokken uniform te stylen. Dat zou dan ook een wijziging in je css betekenen, maar dan kun je het hierna wel hergebruiken over een pagina heen.

Reply:
De class zou ik dan op de bovenliggende div plaatsen denk ik

Reply:
I replaced the `id` that I added in the `css` with a class `redirect` and added it to the `<p>`. I do not believe it has much use to put it in a div as this block is the only place where the particular formatting of `<a>` is desired.